Source/WebCore/Modules/cookie-store/CookieStore.cpp

```diff
@@ -230,6 +230,16 @@ CookieStore::~CookieStore()
     m_mainThreadBridge->detach();
 }
 
+static bool containsInvalidCharacters(const String& str)
+{
+    // The invalid characters are specified at https://wicg.github.io/cookie-store/#set-a-cookie.
+    return str.contains([](UChar c) {
+        return c == 0x003B
```

Comment:
Could all these 3 conditions be on 1 line.

Reply:
Fixed!